doc/architecture.rst

```diff
@@ -369,3 +369,37 @@ is renamed or moved.
 Example:
 
   <oc:id>00000020oc5cfy6qqizm</oc:id>
+  
+Virtual Files
+----------------
+.. note::
+    * This feature is currently only available on ``Windows`` by default. ``Linux`` implementation is experimental and must be enabled by adding ``enableExperimentalOptions=true`` to the ``nextcloud.cfg`` configuration file in the ``App Data`` folder. ``macOS``, at the moment, is using the same backend as ``Linux`` one. It can be enabled with the same ``enableExperimentalOptions`` flag.
```

Comment:
I'd change for "Linux and macOS implementations are experimental [...]". This allows to drop the last sentence of that paragraph and should be more readable.

Reply:
See https://github.com/nextcloud/desktop/pull/3841